setup.py

```diff
@@ -132,9 +132,18 @@ def make_release_tree(self, base_dir, files):
         'Topic :: System :: Filesystems',
         'Topic :: Utilities',
     ],
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, <4',
+    # Gsutil supports Python 2.7, 3.5+
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4',
```

Comment:
Super nit: would it be worth adding an explicit `>=3.5.*` here?

Reply:
We already have a `>=2.7` expression; wouldn't the `>=3.5.*` intersect with that?

Reply:
Oh you're right! Whoop, that makes sense :)